### ISSUE TITLE
Long support?

### ISSUE THREAD

**Issue by delaneyj** [kriasoft/knex-types#1]
For the 64bit types can there be optional usage of [long](https://www.npmjs.com/package/long) support instead of just string?

**Comment by koistya** [comment]
Yep. Would you be able to add it to this list? https://github.com/kriasoft/knex-types/blob/6ad39319a483139639f74f2442dac54b9e81edcd/main.ts#L155-L206
